content/doc/book/blueocean/index.adoc

```diff
@@ -42,7 +42,7 @@ For an overview of content in the Jenkins User Handbook, see
 
 
 [[blue-ocean-overview]]
-== What is Blue Ocean?
+== What is the Blue Ocean?
```

Comment:
The article "the" in that location may mislead the reader of the usage of "Blue Ocean".  It was a project named "Blue Ocean".  I think this should remain as it was before.
```suggestion
== What is Blue Ocean?
```